module-domain/src/main/java/com/yapp/sharefood/food/domain/FoodStatus.java

```diff
@@ -1,5 +1,9 @@
 package com.yapp.sharefood.food.domain;
 
 public enum FoodStatus {
-    SHARED, MINE
+    SHARED, MINE;
+
+    public boolean isShared() {
+        return this.name().equals(SHARED.name());
+    }
```

Comment:
```suggestion
    public boolean isShared() {
        return this == FoodStatus.SHARED;
    }
```
enum은 자체가 singleton 입니다 그래서 이렇게 바꾸는게 어떨까요?